server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRestoreTask.java

```diff
@@ -50,6 +50,12 @@
             this.errorRestoreCount = new AtomicLong();
         }
 
+        AdditionalInformation(User user, AtomicLong successfulRestoreCount, AtomicLong errorRestoreCount) {
```

Comment:
Does it makes sens to serialize / deserialize mutable data?

Shouldn't the AdditionalInformation just be an immutable POJO and rely on the Task for successCount/errorCount tracking?

Reply:
I agree with you.

Reply:
(sorry...)

Reply:
@remk thanks